### PR TITLE
Add multiple formats to in-vitro data pivots

### PR DESCRIPTION
**Pull request by shapiromatron**
Design two different format-styles for in-vitro data-pivots- one where data are aggregated by Endpoint, and one where data are aggregated by Endpoint-group.

### CODE REVIEW

File: project/invitro/exports.py [hunk 1]
```diff
@@ -1,9 +1,28 @@
+from copy import copy
 from django.db.models.loading import get_model
 
 from utils.helper import FlatFileExporter
 
 
-class IVEndpointFlatDataPivot(FlatFileExporter):
```
Comment (JoshAddington) [inline project/invitro/exports.py]: invitro.views - line 42 calls IVEndpointFlatDataPivot, should that be changed to DataPivotEndpoint?

Comment (shapiromatron) [inline project/invitro/exports.py]: I think it's ok; see line 25 below. I just renamed the class and pulled-out some shared helper functions

Comment (JoshAddington) [inline project/invitro/exports.py]: https://github.com/shapiromatron/hawc/blob/master/project/invitro/views.py#L42

Comment (shapiromatron) [inline project/invitro/exports.py]: Oh thanks; missed that one, I'll change.

**Comment by JoshAddington** [comment]
I can't test, as I don't have invitro data available, but following the code looks good.  :+1:

**Pull request merged**